assets/js/store/modules/message.js

```diff
@@ -0,0 +1,24 @@
+export default {
+    namespaced: true,
+    state: {
+      message: ''
+    },
+    mutations: {
+        SET_MESSAGE(state, message ){
+            state.message = message
+        }
+    },
+    actions: {
+        setMessage({ commit }, options) {
+            if (options.type === 'success'){
+                commit('SET_MESSAGE', options.message)
```

Comment:
Let's treat a message object as a whole message, we probably will need this information about the type.